src/AutoPilotPlugins/APM/APMPowerComponent.qml

```diff
@@ -29,6 +29,7 @@ SetupPage {
         Column {
             spacing: _margins
 
+            property Fact armVoltMin:       controller.getParameterFact(-1, "ARMING_VOLT_MIN")
```

Comment:
The name of this parameter changed: https://github.com/ArduPilot/ardupilot/commit/f5f6ec5
We need to handle that by looking for a parameter matching either name. Look at how the RC* and SERVO* parameters are handled.

Reply:
Reference: https://github.com/bluerobotics/qgroundcontrol/blob/master/src/FirmwarePlugin/APM/ArduSubFirmwarePlugin.cc